chainercv/links/model/faster_rcnn/utils/anchor_target_creator.py

```diff
@@ -0,0 +1,203 @@
+import numpy as np
+
+import chainer
+from chainer import cuda
+
+from chainercv.links.model.faster_rcnn.utils.bbox2loc import bbox2loc
+from chainercv.utils.bbox.bbox_iou import bbox_iou
+
+
+class AnchorTargetCreator(object):
+
+    """Assign anchors to the ground-truth targets.
+
+    Assigns anchors to ground-truth targets to train Region Proposal Networks
+    introduced in Faster R-CNN [#]_.
+
+    Bounding regression targets are computed using encoding scheme
+    found in :obj:`chainercv.links.model.faster_rcnn.bbox2loc`.
+
+    .. [#] Shaoqing Ren, Kaiming He, Ross Girshick, Jian Sun. \
+    Faster R-CNN: Towards Real-Time Object Detection with \
+    Region Proposal Networks. NIPS 2015.
+
+    Args:
+        n_sample (int): Number of regions to produce.
+        positive_iou (float): Anchors with IoU above this
+            threshold will be assigned as positive.
+        negative_iou (float): Anchors with IoU below this
+            threshold will be assigned as negative.
+        fg_fraction (float): Fraction of positive regions in the
+            set of all regions produced.
+        loc_in_weight (tuple of four floats): Four coefficients
+            used to calculate loc_in_weight.
```

Comment:
Sorry, I misunderstood. `loc_in_weight_base` and `loc_in_weight` look good to me.